portal/templates/reporting_dashboard.html

```diff
@@ -6,6 +6,9 @@ <h3 class="tnth-headline">{{ _("Reporting Dashboard") }}</h3>
 
     <h2 class="tnth-headline">{{ _("Usage Statistics") }}</h2>
     <div>
+        <p>{{ _("Registrations are collected from the User.registration timestamps of all Users without the Test Role") }}</p>
+        <p>{{ _("Logins are collected from the start_time timestamps of Encounters whose auth_method='password_authenticated'") }}</p>
+        <p>{{ _("Intervention Logins are filtered based on the login Encounter's User's User.interventions") }}</p>
         <table id="usageStatsTable"
```

Comment:
@OptimusRhine once you've made that change we can merge it in - I'm certain it will need more, but I'd like to see these changes in context.